specs/binding-kafka.spec/src/main/resources/META-INF/zilla/kafka.idl

```diff
@@ -501,6 +512,87 @@ scope kafka
             int64 producerId;
             int16 producerEpoch;
         }
+
+        struct KafkaCreateTopic
+        {
+            string16 name;
+            int16 replicationFactor;
+            KafkaPartition[] assignments;
+            KafkaConfig[] configs;
+        }
+
+        struct KafkaCreateTopicsRequestBeginEx
+        {
+            KafkaCreateTopic[] topics;
+            uint8 validateOnly;
```

Comment:
Need `timeout` here too.

Reply:
I thought timeout is something internal to be handled.